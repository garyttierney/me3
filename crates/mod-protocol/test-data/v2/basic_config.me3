profileVersion = "v2"

[mods]
my_mod.path = './my-mod'
my_dll.path = './my-mod/my_dll.dll'
my_profile.path = 'my_profile.me3'
