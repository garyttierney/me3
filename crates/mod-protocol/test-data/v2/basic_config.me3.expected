V2(
    ModProfileV2 {
        supports: None,
        natives: [
            Native {
                inner: ModFile {
                    name: "my_dll",
                    path: "./my-mod/my_dll.dll",
                    enabled: true,
                    optional: false,
                },
                initializer: None,
                load_before: [],
                load_after: [],
            },
        ],
        packages: [
            Package {
                inner: ModFile {
                    name: "my_mod",
                    path: "./my-mod",
                    enabled: true,
                    optional: false,
                },
                load_before: [],
                load_after: [],
            },
        ],
        profiles: [
            ModFile {
                name: "my_profile",
                path: "my_profile.me3",
                enabled: true,
                optional: false,
            },
        ],
        savefile: None,
        start_online: None,
        disable_arxan: None,
    },
)
